wordpress-mobile/WordPress-Android
# Issue/13326 changes for fluxc scanstore suspend function

Parent #13326

This PR adjusts code + tests to start using suspend functions from https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1866

To test:

There is no visual changes, make sure that CI runs tests without issues. 
(Might also want to load scan screen to see nothing has broken)

Merge Instructions:

- Review FluxC PR wordpress-mobile/WordPress-FluxC-Android/pull/1866  merge and create a tag in FluxC develop
- Replace FluxC hash with tag from develop
- Remove the "Not Ready for Merge label"
- Merge this PR

PR submission checklist:
